src/components/common/NameRequestInfo.vue

```diff
@@ -196,4 +383,13 @@ ul.numbered-company-list-items {
     margin-left: -1.5em;
   }
 }
+
+#name-translation-info {
+  display: flex;
+  align-items: baseline;
+
+  .v-input--selection-controls .v-input__slot > .v-label, .v-input--selection-controls .v-radio > .v-label {
```

Comment:
To make this easier to read, I suggest putting the 2 expression on separate lines :)

Also I wonder if an id or class on the label would be cleaner?

Reply:
😆 Sorry, i forgot to update this. I was testing to find the element, will update this!

Reply:
If you're wondering, as I scan/review code, this line jumps out :)

Basically you write great code though -- easy to understand and review! Well done 👍

Reply:
haha... yea, it certainly does 😆 

Thanks a lot!